src/lib/Components/Bidding/Screens/Registration.tsx

```diff
@@ -143,17 +155,34 @@ export class Registration extends React.Component<RegistrationProps, Registratio
 
   createBidder() {
     commitMutation(this.props.relay.environment, {
-      onCompleted: () => {
-        console.log("COMPLETED!")
-      },
-      onError: () => {
-        console.log("ERROR!")
-      },
+      onCompleted: (results, errors) =>
+        isEmpty(errors) ? console.log("COMPLETED!", results) : this.presentErrorResult(errors, msgForGraohQLError),
+      onError: error => this.presentErrorResult(error, msgForNetworkError),
       mutation: bidderMutation,
       variables: { input: { sale_id: this.props.sale.id } },
     })
   }
 
+  presentErrorResult(error, message) {
+    console.error(error)
+
+    this.props.navigator.push({
+      component: BidResultScreen,
```

Comment:
We are re-using the `BidResultScreen` screen, but this doesn't seem right. We should switch that to the new `RegistrationResult` component once https://artsyproduct.atlassian.net/browse/PURCHASE-202 is done. Alternatively, we could add a separate `ErrorScreen` component that only shows an error message, but I'll leave it to whoever works on PURCHASE-202.